src/App.js

```diff
@@ -4,28 +4,34 @@ import Tile from './Tile';
 import DetailTile from './DetailTile';
 import Details from './Details'
 import './App.css';
-import movieData from './movieData'
+import { fetchAllMovies, fetchMovieDetails } from './apiCalls'
 
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movieData: movieData,
-      movieDetails: ""
+      movieData: [],
+      movieDetails: "",
```

Comment:
Does `movieDetails` stay as a string? If so, then this is ok, but if it changes to an object, then you should consider starting this piece of state as an empty object. When data types change when the app is running, it can be very confusing for other developers.

Reply:
It does not so we changed it to be an empty object.